pkgs/data/fonts/kacst/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation {
+  name = "kacst-2.01";
```

Comment:
Seems better to put 2.01 in the version.

Reply:
Thanks, I'll fix that